cookie-banner-rules-list.json

```diff
@@ -3744,9 +3668,9 @@
     },
     {
       "click": {
-        "optIn": "button.corgi__sc-140l2f3-1",
-        "optOut": "button.corgi__sc-1k5bfrh-3",
-        "presence": "div.bJyFEZ"
+        "optIn": ".corgi__sc-140l2f3-0.corgi__sc-140l2f3-1.corgi__sc-1k5bfrh-2.foVfGp.gjMFpW.esJNpe",
+        "optOut": ".corgi__sc-140l2f3-0.corgi__sc-1k5bfrh-3.gIWeVI.pjCEP",
+        "presence": ".corgi__sc-p6yu06-0.UAcHO"
```

Comment:
Instead of relying on these semi-random class names, can we use the `data-testid` attributes instead? For exampel the reject-all button has `data-testid="cookie-wall-reject"`.